src/style.css

```
@@ -0,0 +1,152 @@
+body,
+h3,
+p {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  background: #fff;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+}
+
+.card__logo {
+  width: 160px;
+  height: 134px;
+  padding: 16px 3px 40px;
+}
+
+.card__quantity {
+  font-size: 10px;
+  height: 16px;
+  line-height: 18px;
+}
+
+.name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.card__price {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: rgba(97, 96, 112, 1);
+}
+
+.card__value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.card > a {
+  box-sizing: border-box;
+  text-transform: uppercase;
+  background-color: rgba(0, 172, 220, 1);
+  text-decoration: none;
+  border-radius: 5px;
+  height: 40px;
+  line-height: 40px;
+  text-align: center;
+  color: white;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 4px 0;
+  color: rgba(97, 96, 112, 1);
+}
+
+.card__code {
+  padding-bottom: 16px;
+  height: 14px;
+  line-height: 14px;
+  font-size: 10px;
+}
+
+.card__reviewWrap {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 24px;
+}
+
+.card__priceWrap {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.card > a:hover {
+  text-transform: uppercase;
+  background-color: #fff;
+  text-decoration: none;
+  border: 1px solid rgba(0, 172, 220, 1);
+  border-radius: 5px;
+  color: rgba(0, 172, 220, 1);
+  height: 40px;
+  line-height: 40px;
+}
+
+main {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+main > .card {
+  margin: 8px;
+}
+
+/* stars section */
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 > .stars__star:nth-of-type(n + 5),
+.stars--5 > .stars__star:nth-of-type(n + 1),
+.stars--4 > .stars__star:nth-of-type(n + 2),
+.stars--3 > .stars__star:nth-of-type(n + 3),
+.stars--2 > .stars__star:nth-of-type(n + 4),
+.stars--0 > .stars__star:nth-of-type(n + 6) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  display: flex;
+}
```

```suggestion
.stars--1 > .stars__star:nth-of-type(n + 5),
.stars--5 > .stars__star:nth-of-type(n + 1),
.stars--4 > .stars__star:nth-of-type(n + 2),
.stars--3 > .stars__star:nth-of-type(n + 3),
.stars--2 > .stars__star:nth-of-type(n + 4),
.stars--0 > .stars__star:nth-of-type(n + 6) {
  background-image: url(./images/star-active.svg);
}
```
don't repeat yourself